src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from "react";
```

Comment:
loading ini jg apaa yaa apakah diperluinn?

Reply:
ada di branch loading sebelah mas

Reply:
oh, ga dibuatnya di branch itu aja? di sini di delete aja(?) ato diperluin buat button?

Reply:
eh ini kalo udah terlanjur yaudah gpp ca kalo gt deh, yang penting conflict, tapi in the future branchingnya dari master aja biar kalo ada fitur yang ga mau kebawa tinggal ga usah di merge gt

Reply:
ini casenya waktu awal aku buat mau dijadiin satu branch gtu, terus gajadi dan file loading ini lupa dihapus

Reply:
deleted mas